web/src/compose_call.ts

```diff
@@ -1,7 +1,7 @@
 import {realm} from "./state_data";
 
 export const zoom_token_callbacks = new Map();
-export const video_call_xhrs = new Map<string, JQuery.jqXHR<unknown>>();
+export const video_call_xhrs = new Map<string, JQuery.jqXHR<unknown> | undefined>();
```

Comment:
Why is this type change correct? I see us doing `delete` to remove items from this, not storing `undefined` in it.

(Generally, the answer to this class of question should be in the paragraph portion of the commit message for this sort of refactoring commit)